### DISCUSSION TITLE
Lets add a List component !!!

### DISCUSSION THREAD

**Discussion by sanskarg348** [Atri-Labs/atrilabs-engine#463] [Discuss: New Feature Ideas]
It can help the developer experience and accelerate the development if we have a list component !!!

**Comment by cruxcode** [comment]
Yes, it will be really helpful. It might be a bit challenging because we will have to give it support at three places:
1. The visual editor
2. The python code
3. The generated code

The way it will work is that:
1. You drag and drop a List component.
2. You drag and drop another component inside the List component you dropped in the earlier step.
3. We will repeat the component an `x` number of times where x is a custom field.
